charts/karpenter/values.yaml

```diff
@@ -162,3 +162,4 @@ clusterEndpoint: ""
 aws:
   # -- The default instance profile to use when launching nodes on AWS
   defaultInstanceProfile: ""
+  enablePodENI: ""
```

Comment:
Would it be better to just set the default as `false` in the helm values for visibility since we know what the default is? My understanding is that this is consistent with how other projects handle boolean helm values.

Reply:
Sure, we certainly can do that.

Reply:
Still looks like this is a string value

Reply:
I'm pretty sure it has to be a string.

Reply:
oh, nope, I'm wrong.  K8S expects the value to be a string, but we handle that with 
```
{{ quote .Values.aws.enablePodENI }}
```

Reply:
Fixed now.